### PR TITLE
Trigger change event on hidden input when image id changes

### PR DESCRIPTION
**Pull request by astagi**

### CODE REVIEW

**Comment by yakky** [comment]
LGTM
@stefanfoulis you ok with this?

File: filer/static/filer/js/popup_handling.js [hunk 1]
```diff
@@ -9,10 +9,13 @@
 		var txt_name = name + '_description_txt';
 		var clear_name = name + '_clear';
 		var elem = document.getElementById(name);
+		var old_id = elem.value;
 		document.getElementById(name).value = chosenId;
 		document.getElementById(img_name).src = chosenThumbnailUrl;
 		document.getElementById(txt_name).innerHTML = chosenDescriptionTxt;
 		document.getElementById(clear_name).style.display = 'inline';
+		if (old_id != chosenId)
```
Comment (yakky) [inline filer/static/filer/js/popup_handling.js]: current code style requires to always use brackets

Comment (astagi) [inline filer/static/filer/js/popup_handling.js]: Updated! sorry :)

Comment (yakky) [inline filer/static/filer/js/popup_handling.js]: Thanks

**Pull request merged**